python/sdk/prelude_sdk/models/codes.py

```diff
@@ -45,26 +45,24 @@ def _missing_(cls, value):
 
 class ExitCode(Enum):
     MISSING = -1
-    REMOVED = 0
-    ERROR = 1
-    MALFORMED_VST = 2
-    PROCESS_KILLED_1 = 9
-    PROCESS_KILLED_2 = 15
-    PROTECTED = 100
-    UNPROTECTED = 101
-    TIMEOUT = 102
-    CLEANUP_ERROR = 103
+    UNKNOWN_ERROR = 1
+    MALFORMED_TEST = 2
+    BEHAVIOR_PREVENTED_1 = 9
+    BEHAVIOR_PREVENTED_2 = 15
+    CHECK_COMPLETED = 100
+    NOT_PREVENTED = 101
+    TIMED_OUT = 102
+    FAILED_CLEANUP = 103
     NOT_RELEVANT = 104
-    QUARANTINED_1 = 105
-    OUTBOUND_SECURE = 106
-    EXPLOIT_PREVENTED = 107
-    NO_TEST = 108
-    IS_RELEVANT = 109
-    FALSE_POSITIVE = 110
-    ENDPOINT_BLOCKED = 126
-    QUARANTINED_2 = 127
-    PROCESS_KILLED_3 = 137
-    UNEXPECTED = 256
+    SIGNATURE_PREVENTED_1 = 105
+    BLOCKED_AT_PERIMETER = 106
+    BEHAVIOR_PREVENTED_3 = 107
+    TEST_UNAVAILABLE = 108
+    INCORRECTLY_BLOCKED = 110
+    BEHAVIOR_PREVENTED_4 = 126
+    SIGNATURE_PREVENTED_2 = 127
+    OUT_OF_MEMORY = 137
+    UNEXPECTED_ERROR = 256
 
```

Comment:
we also have a transform() method  that BE uses:
```

    def transform(self, test):
        if test.unit == 'health' and self != ExitCode.PROTECTED:
            return ExitCode.FALSE_POSITIVE
        return self
```

Reply:
I added it with the updated values